package.json

```diff
@@ -13,10 +13,13 @@
     "async": "^2.6.0",
     "bcrypt": "^1.0.3",
     "cassandra-driver": "^3.3.0",
+    "chai": "^4.1.2",
+    "chai-http": "^3.0.0",
     "express": "^4.16.2",
```

Comment:
why is chai a dependency when mocha is a dev dependency

Reply:
Yeah should be test

Reply:
will move it to dev dependencies